### ISSUE TITLE
Watch Directory Issue with Path Spaces

### ISSUE THREAD

**Issue by iamfreaky** [rakshasa/rtorrent#834]
I get the Following error with a watch directory 

Scheduled command failed: watch_directory_1: Junk at end of input.

schedule2 = watch_directory_1, 5, 5, "load.start_verbose=/media/Videos/1_Watch/5_Test Test/*.torrent,d.set_directory=/media/Videos/1_Watch/5_Test Test/"

while this works without any Problems

schedule2 = watch_directory_1, 5, 5, "load.start_verbose=/media/Videos/1_Watch/5_TestTest/*.torrent,d.set_directory=/media/Videos/1_Watch/5_TestTest/"

So basically the spaces between words are fucking things up.

**Comment by vineryap** [comment]
I have the same issue.
Tried escaping it with `\` but no luck.
anyone know how to solve this?
